frappe_er_generator/frappe_er_generator/er_generator.py

```diff
@@ -132,11 +136,15 @@ def get_table(data, link_list, doctypes):
             else:
                 table_element_list.append(
                     f'<tr><td port="{field.get("fieldname")}">{field.get("label")}</td></tr>')
-        if field.get("fieldtype") == "Link":
+        if field.get("fieldtype") == "Link" and (amended_from or field.get("fieldname") != "amended_from"):
```

Comment:
1. What if I want to show Linkage of amended_from field by default , amended_from should be True by default.
2. You need to check field.get('options') !== data.doctype , as in some cases fieldname can be different

Reply:
1. Got it, it won't change original functionality now.
2. It was intended to only ignore amended_from. Modified to omit self-links or any link name passed by itself or in an iterable.